guide/how-it-works/custom-spending-conditions.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: guide
+title: Custom Spending Conditions
+description: An overview about advanced capabilities that enable users to create more flexible bitcoin wallets.
+nav_order: 12
+parent: How it works
+permalink: /guide/how-it-works/custom-spending-conditions/
+main_classes: -no-top-padding
+image: /assets/images/guide/how-it-works/custom-spending-conditions/header.jpg
+image_base: /assets/images/guide/how-it-works/custom-spending-conditions/
+
+---
+
+<!--
+
+Editor's notes
+
+This page focuses on the advanced wallet configurations which are enabled by Miniscript.
+
+Header image source:
+https://www.figma.com/file/qzvCvqhSRx3Jq8aywaSjlr/Bitcoin-Design-Guide-Illustrations-CO?type=design&node-id=3324%3A24043&mode=design&t=GbcNUdc0VUnSOQi2-1
+
+-->
+
+{% include picture.html
+   image = "assets/images/guide/how-it-works/custom-spending-conditions/header.jpg"
+   retina = "assets/images/guide/how-it-works/custom-spending-conditions/header@2x.jpg"
+   alt-text = ""
+   width = 1600
+   height = 800
+   layout = "full-width"
+%}
+
+
+# Custom Spending Conditions
+{:.no_toc}
+---
+
+Spending conditions encode the rules that need to be followed by a wallet to spend funds. Most bitcoin wallets have only one spending condition. For example, in a single-key wallet there is only one key that can sign transactions. Or, in a 2-of-3 multi-key wallet, you need any two of the three keys to sign transactions.
+
+In addition to these standard configurations it is also possible to create more flexible wallets by defining custom spending conditions. There is a wide variety of use cases, where this flexibility can be helpful, like personal or family self-custody, inheritance or company treasuries.
+
+## Use case examples
+
+#### Self-custody
+
+Custom spending conditions can be helpful to decrease the risk of losing funds in personal self-custody context. In a savings wallet context where you are the only key holder, you could unlock a recovery spending condition after a certain time of wallet inactivity. This condition could contain an additional key held in a different location. Such a setup does not decrease the wallet security, because the second key is locked under normal circumstances of using the wallet. 
+
+In a multi-key context, the recovery condition could require only one instead of two keys to spend funds after 6 months of inactivity. This will help mitigate the risk of loss of funds in case one key holder loses a key.
+
+#### Company treasuries
+
+Custom spending conditions can also be useful in an organizational context. Companies need to make sure that they retain access to funds even if one or more employees lose their keys, are compromised or try to obstruct the deployment of funds. Or they might want to implement company-specific processes for corporate spending. For example, they would want to create a rule that the CFO must sign transactions for treasury management. 
```

Comment:
```suggestion
Custom spending conditions can also be useful in an organizational context. Companies need to make sure that they retain access to funds even if one or more employees lose their keys. It can also be to ensure that funds are not compromised or the deployment of funds is not obstructed. Or they might want to implement company-specific processes for corporate spending. For example, they would want to create a rule that the CFO must sign transactions for treasury management. 
```

Reply:
Counter proposal 🙂:

```suggestion
Custom spending conditions can also be useful in an organizational context. Companies need to make sure that they retain access to funds even if one or more employees lose their keys. Not only that, employees could also be compromised by a third party or try to obstruct the deployment of funds for their own benefit. Another use case is that a company needs to implement company-specific processes for corporate spending. For example, they would want to create a rule that the CFO must sign transactions for treasury management. 
```

Reply:
Thumbs up for the counter proposal :-)